webpack/js/main.js

```diff
@@ -1,74 +1,112 @@
 $(document).ready(function () {
-    $(window).scroll(function () {
-        if ($(this).scrollTop() > 500) {
-            $('#back-to-top').fadeIn();
-        } else {
-            $('#back-to-top').fadeOut();
-        }
-    });
+  $(window).scroll(function () {
+    if ($(this).scrollTop() > 500) {
```

Comment:
Would using a hardcoded value here affect how this behaves on different screen sizes?

Reply:
Yeah, it won't. Will find an optimal way.